packages/transitive-overlay/src/index.tsx

```diff
@@ -139,7 +145,9 @@ const TransitiveCanvasOverlay = ({
                       type: "Feature",
                       properties: {
                         type: "street-edge",
-                        color: modeColorMap[segment.type] || "#008",
+                        color: ignoreRouteColor
+                          ? segment?.route_color || defaultColorOverride
+                          : modeColorMap[segment.type] || "#008",
```

Comment:
Is this conditional flipped?

```suggestion
                        color: ignoreRouteColor
                          ? modeColorMap[segment.type] || "#008"
                          : segment?.route_color || defaultColorOverride,
```

Reply:
I'm also a little confused as to when `defaultColorOverride` is used.